api/v1/openapi.yaml

```diff
@@ -3120,6 +3120,7 @@ definitions:
             - HostPort
             - LoadBalancer
             - LocalRedirect
+            - NONE
```

Comment:
Instead of `NONE` why don't we use `Headless` which would be more explicit?

Reply:
Good point :heavy_check_mark: